scheduler-jobs/deploy_data_specific_image

```diff
@@ -82,10 +82,14 @@ for ((i=0;i<$CONFIG_LIST_LENGTH;i++)); do
 
     echo "Extracting ${FULL_URL} to ${FULL_DEST_DIR}"
 
-    # Download the data from github and copy it to the destination directory
-    # TODO: Handle renamed or deleted notebooks, since this command only copies the content of the source_dir in the dest_dir
-    # For now, any other file found in the dest_dir are not handled and remain in its location.
-    svn export --force $FULL_URL $FULL_DEST_DIR
+    # Download the data from github into tmp directory
+    svn export --force $FULL_URL tmp
```

Comment:
why not `/tmp`?

Reply:
I'd say `/tmp/notebooks` to not mixed up with other stuff already in `/tmp/`. But yes, it's usually better to use absolute path.  Just `tmp` not sure where it is.

Reply:
I agree with you, It should be `/tmp/notebooks` and because `/tmp/` already exists and might contain some other stuff, it will only remove '/tmp/notebooks'

Reply:
And maybe clone under `/tmp/notebooks/{random_hash}`, since conceptually what you get through `svn` is one of the notebooks item, not the only one (reason why it's in a loop). Also, that would push the `rm -rf /tmp/notebooks` instruction out of the loop.